pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -642,7 +642,22 @@ func BuildKubernetesMasterConfig(
 	return kmaster, nil
 }
 
-func DefaultOpenAPIConfig() *openapicommon.Config {
+func DefaultOpenAPIConfig(config configapi.MasterConfig) *openapicommon.Config {
+	securityDefinitions := spec.SecurityDefinitions{}
```

Comment:
I am not sure where to source the right info, for example about flow type and scopes, and no other code in the origin repo uses the oauth2 type yet, it seem any outh based on any kind of token is defined just as an apikey bearer token...